### ISSUE TITLE
Token Status Page Skeleton

### ISSUE THREAD

**Issue by daltonfury42** [SimplQ/simplQ-frontend#282]
The token status page has to be updated as per the [finalised design](https://xd.adobe.com/view/ad1db074-03bf-45b1-537b-98d9d524ec82-db2c/screen/e87838cd-febc-4f37-a68a-9daa0d6b207b)

![image](https://user-images.githubusercontent.com/11256376/94916015-7d2e7180-04cb-11eb-99cf-c995de5b5148.png)

The basic component structure consists of 4 major regions:

<img width="1211" alt="Screenshot 2020-10-02 at 4 23 35 PM" src="https://user-images.githubusercontent.com/11256376/94916857-13af6280-04cd-11eb-8640-932f880cbc09.png">

1) Header
2) TokenNumber
3) TokenStatus
4) SidePanel

This task is to get the framework correct. We can style each component independently in follow up issues.

[src/components/pages/Status](https://github.com/SimplQ/simplQ-frontend/tree/new-frontend/simplq/src/components/pages/Status) can have 

1) One index.jsx that ties all the components together, it's would roughly render 
```
<Header>
<div className="main-content"> // Would be a flexbox with flex-wrap to accommodate  smaller screen
  <TokenNumber> <TokenStatus> <SidePanel>
</div> 
```

cc: @maaverik What do you think?

**Comment by deprov447** [comment]
@maaverik @daltonfury42 Please let me know If I can contribute any component.

**Comment by daltonfury42** [comment]
@maaverik have you started working on this? Else do you want to let @deprov447 take this up?

We want to do restructure the page according to the new design, creating just the component hierarchy. Then we can split the styling and functionality of each component into independent tasks and split them amongst us.

**Comment by maaverik** [comment]
Yeah ,I've made the skeleton of sorts in #288. @deprov447, you can take over once this is merged.

**Comment by maaverik** [comment]
We'll make separate issue for each of the subcomponents and you can work on whichever you want.

**Comment by maaverik** [comment]
@deprov447 take a look at #291 or #292 and you can get started on finishing this page.
